Userland/Libraries/LibGUI/Calendar.cpp

```diff
@@ -467,10 +468,19 @@ void Calendar::paint_event(GUI::PaintEvent& event)
                     m_tiles[0][i].width,
                     m_tiles[0][i].height);
                 m_tiles[0][i].rect = tile_rect.translated(frame_thickness(), frame_thickness());
-                if (m_tiles[0][i].is_hovered || m_tiles[0][i].is_selected)
-                    painter.fill_rect(tile_rect, palette().hover_highlight());
-                else
-                    painter.fill_rect(tile_rect, palette().base());
+
+                Color background_color = palette().base();
+
+                if (m_tiles[0][i].is_hovered || m_tiles[0][i].is_selected) {
+                    background_color = palette().hover_highlight();
+                } else if (is_weekend) {
+                    if (palette().is_dark())
+                        background_color = palette().base().lightened(1.2f);
+                    else
+                        background_color = palette().base().darkened(0.9f);
```

Comment:
Can we tweak this color a bit? I had to put my face all the way up to my screen to see the difference between this and the highlight color with a light theme :)

Reply:
This seems a bit more distinguishable to me:
```c++
else if (is_weekend) {
    background_color = palette().gutter();
}
```